policyengine_us/parameters/gov/states/co/tax/income/credits/eitc/match.yaml

```diff
@@ -0,0 +1,12 @@
+description: CO matches this percent of the federal EITC
+values:
+  0000-01-01: .1
```

Comment:
Probably not correct...it might be in the legal code, but if not you could do 2021 instead. Use the earliest year you're confident in.